test/react/server_rendering/manifest_container_test.rb

```diff
@@ -1,8 +1,6 @@
 require "test_helper"
 
-# Rails 3.x doesn't have `application.assets_manifest=`
-# I don't think we have to support Rails 3 + Sprockets 3 anyways!
-if Rails::VERSION::MAJOR > 3
+if React::ServerRendering::ManifestContainer.compatible?
```

Comment:
Yes, I agree -- better to keep the tests simple so we'll see if a future change causes them to break! 

Alternatively, you could add unit tests for `.compatible?`